src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/OracleTranslator.java

```diff
@@ -294,6 +297,29 @@ public String translate(DbColumn c) {
         }
     }
 
+    @Override
+    public String translate(final DbColumnType type) {
+        switch (type) {
+            case BOOLEAN:
+                return "NUMBER(1,0)";
+
+            case DOUBLE:
+                return "BINARY_DOUBLE";
+
+            case INT:
+                return "INT";
+
+            case LONG:
+                return "NUMBER(19,0)";
+
+            case STRING:
+                return format("VARCHAR(%s)", properties.getProperty(VARCHAR_SIZE));
```

Comment:
We should not set the size. Can we just use `VARCHAR`?

Reply:
Nop. In Oracle and DB2 you are forced to specify a size.